cdap-common/src/main/java/co/cask/cdap/common/exception/ConflictException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.common.exception;
+
+/**
+ * Thrown when there was a conflict.
+ */
+public class ConflictException extends Exception {
```

Comment:
We don't yet use this, right? Is the idea that wherever we check for ``CONFLICT``, we will instead throw this exception in later PRs?